packages/@okta/vuepress-site/guides/sign-into-web-app/define-callback/aspnet/define-route.md

```diff
@@ -0,0 +1,3 @@
+The Okta ASP.NET SDK will configure and host this route for you. By default, the route is hosted at `/authorization-code/callback`.
+
+You'll install the SDK later in [Add and Configure Packages](../-/configure-packages/).
```

Comment:
You install the SDK later in [Add and Configure Packages](../-/configure-packages/).

Reply:
Future tense ("you will") makes sense to me here since it is a future action. If there's some grammar rule I am goofing up, ping me 🙂